app/models/grda_warehouse/tasks/task_instrumentation.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'memory_profiler'
```

Comment:
Maybe should move this inside a conditional to avoid including it in production env.